### ISSUE TITLE
Symbol not found with Ruby 2.2.0p0

### ISSUE THREAD

**Issue by mcoms** [nagachika/ruby-coreaudio#18]
Hi, thanks for this gem which I've been using on OS X Yosemite and Ruby 2.1.5 successfully.

On Ruby 2.2.0, unfortunately I encounter this crash when calling `read` on an input buffer.

```
dyld: lazy symbol binding failed: Symbol not found: _rb_thread_blocking_region
  Referenced from: /Users/nick/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/extensions/x86_64-darwin-14/2.2.0-static/coreaudio-0.0.10/coreaudio/coreaudio_ext.bundle
  Expected in: flat namespace

dyld: Symbol not found: _rb_thread_blocking_region
  Referenced from: /Users/nick/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/extensions/x86_64-darwin-14/2.2.0-static/coreaudio-0.0.10/coreaudio/coreaudio_ext.bundle
  Expected in: flat namespace
```

**Comment by nagachika** [comment]
Thank you for your report!

I've just remember that rb_thread_blocking_region should be replaced with rb_thread_call_without_gvl family.

https://github.com/ruby/ruby/blob/ruby_2_2/NEWS#L271-L273

I'll handle this.

**Comment by nagachika** [comment]
I've just released coreaudio-0.0.11.gem with fix for this issue.
https://rubygems.org/gems/coreaudio/versions/0.0.11
Please try it!

**Comment by mcoms** [comment]
Works perfectly, thanks! :+1:
